### PR TITLE
Update CI config

### PR DESCRIPTION
**Pull request by ijjk**
### Description

As discussed this updates our CI config to leverage new runners.

### CODE REVIEW

**Review by sokra — APPROVED** [review]

**Review by ForsakenHarmony — APPROVED** [review]

**Review by kdy1 — APPROVED** [review]

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -320,59 +320,8 @@ jobs:
...
-          key: prysk-venv-${{ matrix.os.name }}
-
-      - name: Integration Tests
-        run: turbo run test --filter=turborepo-tests-integration --color --env-mode=strict --token=${{ secrets.TURBO_TOKEN }} --team=${{ vars.TURBO_TEAM }}
-        env:
-          EXPERIMENTAL_RUST_CODEPATH: true
-
-  # This is the same as turborepo_integration, but it runs on windows.
-  # Separate entry so it's not required for merge.
-  turborepo_integration_windows:
```
Comment (mehulkar) [inline .github/workflows/test.yml:352]: Please bring this back!

**Review by mehulkar — CHANGES_REQUESTED** [review]

File: .github/workflows/test.yml (hunk 1 above)
Comment (ForsakenHarmony) [reply .github/workflows/test.yml]: done, don't know why it was removed

Comment (mehulkar) [reply .github/workflows/test.yml]: probably a merge, there's been a lot of activity in this file

**Review by mehulkar — APPROVED** [review]

File: .github/workflows/test.yml [hunk 2]
```diff
@@ -312,6 +324,54 @@ jobs:
...
+    needs: [determine_jobs, build_turborepo]
+    if: needs.determine_jobs.outputs.turborepo_integration == 'true'
+    timeout-minutes: 30
+    runs-on: ${{ matrix.os.runner }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os:
+          - name: windows
+            runner: windows-latest
```
[the remark this answers is not in the document]
Comment (ForsakenHarmony) [reply .github/workflows/test.yml]: fixed

**Review by mehulkar — APPROVED** [review]

**Pull request merged**